01_install_requirements.sh

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env bash
 set -ex
 
-source common.sh
```

Comment:
We probably shouldn't remove this, because it sets up logging - we'll want to retain that, particularly in CI:

https://github.com/openshift-metalkube/dev-scripts/blob/master/common.sh#L121..L129

Perhaps we can move that to a common_logging.sh or similar as I mentioned in https://github.com/openshift-metalkube/dev-scripts/pull/281 ?

Reply:
Yea I think that's reasonable - updated